README.md

```diff
@@ -1,6 +1,11 @@
 ![Google Cloud Platform Logo](https://cloud.google.com/_static/images/gcp-logo.png)
 # Google Cloud Tools for IntelliJ plugin
-![Build Status](https://storage.googleapis.com/cloud-tools-for-java-kokoro-build-badges/intellij-ubuntu-master-orb.png)
+
+|  | Build Status | 
+| :--- | :---: |
+| Ubuntu | ![Build Status](https://storage.googleapis.com/cloud-tools-for-java-kokoro-build-badges/intellij-ubuntu-master-orb.png) |
```

Comment:
are these png's dynamic based on the build status? just asking because I see the ubuntu one as failing, but the other as green